tests/framework/PostProcessors/BasicStatistics/gold/basicStatisticsGeneral/checkIfMatrixCanBeComputed_basicStatPP_dump.csv

```diff
@@ -1,2 +1,2 @@
 skew_x,vc_x,percentile_5_x,percentile_95_x,mean_x,kurt_x,median_x,max_x,min_x,samp_x,var_x,sigma_x
-0.11423075075,0.188785800293,5.91996682062,8.1183566722,7.28508256545,-3.67853432603,6.33133089078,8.77067587821,5.91996682062,4.0,1.89150549387,1.37532014232
```

Comment:
Why are there changes in basic statistics results?

Reply:
as COB Chair, I approve the modification/regolding of this test.

Thank you Josh for pointing this out.